src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleKeyUp);
```

Comment:
Consider using arrow functions in the event listener to automatically bind 'this'.